app/controllers/clearance/passwords_controller.rb

```diff
@@ -72,7 +72,8 @@ def password_reset_params
       ActiveSupport::Deprecation.warn %{Since locales functionality was added, accessing params[:user] is no longer supported.}
       params[:user][:password]
     else
-      params[:password_reset][:password]
+      reset_params = params[:password_reset] || {}
+      reset_params[:password]
```

Comment:
Ruby 2.3 introduces the [dig](http://ruby-doc.org/core-2.3.0_preview1/Hash.html#method-i-dig) method that could be used here. I'm guessing clearance supports older versions of ruby though.

There is [a gem](https://github.com/Invoca/ruby_dig) that adds the method to older versions but I think this is fine as is.